app/test/e2e/teams.spec.ts

```diff
@@ -130,55 +136,58 @@ describe('Teams', () => {
   })
 
   describe('Add Team', () => {
-    test('should be able to add a new team', async ({ page }) => {
-      // Mock api
-      await page.route(`**/api/teams`, async (route) => {
-        await route.fulfill({
-          status: 201,
-          contentType: 'application/json',
-          body: JSON.stringify([])
-        })
-      })
-
-      await page.route(`**/api/user/search?address=*`, async (route) => {
+    const addMember = async (page: Page, userIndex: number) => {
+      await page.route(`**/api/user/search*`, async (route) => {
         await route.fulfill({
           status: 200,
           contentType: 'application/json',
           body: JSON.stringify({
             success: true,
-            users: users
+            users: Array(users[userIndex])
           })
         })
       })
-
-      await page.goto('http://localhost:5173/teams')
-      await page.click('div[data-test="add-team"]')
-
       // Fill form
       await page.fill('[data-test="team-name-input"]', 'Team 1')
       await page.fill('[data-test="team-description-input"]', 'Description of Team 1')
-      await page.click('[data-test="member-address-input"]')
-      await page.keyboard.type('0x7')
-      await page.keyboard.up('Enter')
+      await page.fill('input[data-test="share-name-input"]', 'Testing')
+      await page.fill('input[data-test="share-symbol-input"]', 'TST')
+      await page.click(`input[data-test="member-${userIndex}-name-input"]`)
+      await page.keyboard.type(`Local ${userIndex + 1}`)
 
```

Comment:
```suggestion
      await page.fill('[data-test="share-name-input"]', 'Testing')
      await page.fill('[data-test="share-symbol-input"]', 'TST')
      await page.click(`[data-test="member-${userIndex}-name-input"]`)
      await page.keyboard.type(`Local ${userIndex + 1}`)
```

You don't need to add the label befor data-test attribute